pkg/ipcache/kvstore.go

```
@@ -247,6 +250,8 @@ func upsertIPNetsToKVStore(prefixes []*net.IPNet, identities []*identity.Identit
 					}).Error("Failed to clean up CIDR->ID mappings")
 				}
 			}
+
+			return
```

Good catch, this could previously hide errors.